Zend/tests/object_array_cast.phpt

```diff
@@ -14,11 +14,11 @@ var_dump($obj);
 ?>
 --EXPECT--
 object(stdClass)#1 (3) {
-  [0]=>
```

Comment:
PHP's weak typing probably helps here. But, yes, there is the possibility someone relies on this.